runtime/jcl/common/system.c

```diff
@@ -414,9 +414,12 @@ jstring getEncoding(JNIEnv *env, jint encodingType)
 			UDATA handle = 0;
 			J9JavaVM * const vm = ((J9VMThread*)env)->javaVM;
 			char dllPath[EsMaxPath];
+			UDATA written = 0;
 			PORT_ACCESS_FROM_ENV(env);
 			/* libjava.[so|dylib] is in the jdk/lib/ directory, one level up from the default/ & compressedrefs/ directories */
```

Comment:
btw this comment only applies to jdk11 and not jdk8, where the libjava.so is in the `lib/<arch>` directory

Reply:
and since there is a `#if defined(J9VM_JCL_SE11)` just above, this is appropriate :)